### ISSUE TITLE
Dependencies are broken/incomplete, project deployment is hard to follow 

### ISSUE THREAD

**Issue by skyler14** [NVIDIA/radtts#4]
Requirements for this project and incomplete, you run into errors sayin torch, lmdb, no python version is specified in the readme, matplotlib gives errors and numpy version conflicts with other libraries when you run `pip install -r requirements`

If this is supposed to be only run inside a preconfigured NGC environment or some preconfigured image, and  if so can that be specified in the readme (along with python version)

Finally, since there are so many files and configurations expected between these different steps can the readme also include a table of all these files are. A notebook pulling all these together to do a step by step deploy would with concrete example would be great too, and I'd be happy to help put something like that together if I had a table or some kind of clearer reference in the readme about how all this should be organized.

**Comment by rafaelvalle** [comment]
Thank you for your feedback.
The requirements file has been updated and a Dockerfile has been added.

Please follow the README for training and inference guidelines. 
Please open a new ticket if you have specific questions.
